### PR TITLE
Added KBD6X Vimwarrior HHKB TOFU Personal Keymap

### PR DESCRIPTION
**Pull request by devinceble**
Added KBD6X Vimwarrior HHKB TOFU Personal Keymap

## Description
Simple Keymap for my workflow base ok HHKB Keymap

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/kbdfans/kbd6x/keymaps/vimwarrior_hhkb_tofu/readme.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Vimwarrior HHKB Tofu Keymap
+
+Build Hex File:
+
+    make kbdfans/kbd6x:vimwarrior_hhkb_tofu
+
+Flash Keyboard
+
+    make kbdfans/kbd6x:vimwarrior_hhkb_tofu:dfu
```
Comment (drashna) [inline keyboards/kbdfans/kbd6x/keymaps/vimwarrior_hhkb_tofu/readme.md]: Actually, we've recently added a universal `:flash` target, and would prefer that it be used: 

```suggestion
    make kbdfans/kbd6x:vimwarrior_hhkb_tofu:flash
```

Comment (devinceble) [reply keyboards/kbdfans/kbd6x/keymaps/vimwarrior_hhkb_tofu/readme.md]: Noted. I will replace it.

**Comment by drashna** [comment]
Also, for the keymaps, in general, we do like to keep them matching to your github account name, so they're easier to track ownership.

File: keyboards/kbdfans/kbd6x/keymaps/vimwarrior_hhkb_tofu/readme.md (hunk 1 above)
Comment (devinceble) [reply keyboards/kbdfans/kbd6x/keymaps/vimwarrior_hhkb_tofu/readme.md]: WARNING: This board's bootloader is not specified or is not supported by the ":flash" target at this time.

Comment (devinceble) [reply keyboards/kbdfans/kbd6x/keymaps/vimwarrior_hhkb_tofu/readme.md]: Many thanks will update it.

**Comment by devinceble** [comment]
Why does old keymaps on dz60 failing?

**Comment by noroadsleft** [comment]
> Why does old keymaps on dz60 failing?

A couple of them compile in our CI system to a size that's larger than the 32u4's capacity, and one seems to have some sort of RGB configuration error. None of these are related to your changes.

**Pull request merged**